README.rst

```diff
@@ -3,19 +3,19 @@ Go rules for Bazel_
 
 .. Links to external sites and pages
 .. _//tests/core/cross: https://github.com/bazelbuild/rules_go/blob/master/tests/core/cross/BUILD.bazel
-.. _Avoiding conflicts: proto/core.rst#avoiding-conflicts
+.. _Avoiding conflicts: proto/core.md#avoiding-conflicts
```

Comment:
seems like a mistake to touch proto/ paths.

just do manual review of all changed lines, I don't think it's worth writing any automation since we'll be done with this soon